animations/actions/seek

```diff
@@ -0,0 +1,12 @@
+{
+"action:seek": [
+    {
+      "trigger": "perception-check",
+      "preset": "onToken",
+      "file": "jb2a.eyes.01.dark_green.single.0",
+      "options": {
+        "sound": {},
+        "scaleToObject": 0.5
+      }
+    }
```

Comment:
Invalid JSON. You forgot a `]` there.

Please check animations in an editor first.